server/services/applicationService.test.ts

```diff
@@ -117,6 +117,34 @@ describe('ApplicationService', () => {
     })
   })
 
+  describe('getAllByOrigin', () => {
+    const token = 'SOME_TOKEN'
+
+    it('fetches all applications for a given applicationOrigin', async () => {
+      const applications = applicationSummaryFactory.buildList(3)
+
+      const paginatedResponse = paginatedResponseFactory.build({
+        data: applications,
+        totalPages: '50',
+        totalResults: '500',
+        pageNumber: '2',
+      }) as PaginatedResponse<Cas2v2ApplicationSummary>
+
+      applicationClient.getAllByOrigin.mockResolvedValue(paginatedResponse)
+
+      const result = await service.getAllByOrigin(token, 'prisonBail', 2)
+
+      expect(result.data).toEqual(applications)
+      expect(result.pageNumber).toEqual('2')
+      expect(result.pageSize).toEqual('10')
+      expect(result.totalPages).toEqual('50')
+      expect(result.totalResults).toEqual('500')
```

Comment:
Just wondering why you don't just `expect(result).toEqual(paginatedResponse)`? If there are additional fields, you could use `expect.objectContaining()`